Plugins/Directory.Build.targets

```diff
@@ -0,0 +1,15 @@
+<Project>
+<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(SolutionDir)'))" />
+  <PropertyGroup>
+    <RunPostBuildEvent>OnOutputUpdated</RunPostBuildEvent>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' And '$(ProjectName)' != 'GitUIPluginInterfaces'">
+    <PostBuildEvent>set GitExtOutputFolder=$(SolutionDir)GitExtensions\$(OutDir)Plugins
```

Comment:
❗️ Don't use `<PostBuildEvent>`. It does not tie into the build up-to-date check or `/t:Clean`.